packages/components-react/projects/nextjs/app/components-ready.tsx

```diff
@@ -0,0 +1,12 @@
+'use client';
+
+import { componentsReady } from '@porsche-design-system/components-react';
+import type { FC, ReactNode } from 'react';
+import { useEffect } from 'react';
+
+export const ComponentsReady: FC<{ children: ReactNode }> = ({ children }): JSX.Element => {
```

Comment:
Perhaps rename component to something more generic, like `Providers` with proper typing (not sure where the `FC` stuff is coming from) and then apply `PorscheDesignSystemProvider` within.
```
export const Providers = ({ children }: PropsWithChildren<{}>): JSX.Element => {
```